frontend/public/components/catalog/catalog-page.tsx

```diff
@@ -411,7 +437,14 @@ export const Catalog = connectToFlags<CatalogProps>(
     coFetch('/api/helm/charts/index.yaml').then(async (res) => {
       const yaml = await res.text();
       const json = safeLoad(yaml);
-      setHelmCharts(json.entries);
+      let kubernetesVersion: string = '';
+      await k8sVersion()
+        .then((response) => {
```

Comment:
I personally would prefer not to mix async/await with .then/.catch. Could you use a try catch block here instead? What do you think?

Reply:
added try-catch block